contracts/libraries/TwabLib.sol

```diff
@@ -311,15 +313,17 @@ library TwabLib {
     /// @return New TWAB that was recorded.
     function _computeNextTwab(
         ObservationLib.Observation memory _currentTwab,
-        uint256 _currentBalance,
+        uint224 _currentBalance,
         uint32 _time
     ) private pure returns (ObservationLib.Observation memory) {
         // New twab amount = last twab amount (or zero) + (current amount * elapsed seconds)
         return
             ObservationLib.Observation({
-                amount: (uint256(_currentTwab.amount) +
-                    (_currentBalance * (_time.checkedSub(_currentTwab.timestamp, _time))))
-                    .toUint208(),
+                amount: uint256(
+                    _currentTwab.amount +
+                        _currentBalance *
+                        (_time.checkedSub(_currentTwab.timestamp, _time))
+                ).toUint224(),
```

Comment:
do we need to cast here?  current balance is uint224 and currentTwab.amount is uint 224.  I think we can skip the cast